include/openssl/evp.h

```diff
@@ -944,24 +940,28 @@ OPENSSL_EXPORT int EVP_PKEY_kem_check_key(EVP_PKEY *key);
 
 // PQDSA specific functions.
 
-#ifdef ENABLE_DILITHIUM
 // EVP_PKEY_CTX_pqdsa_set_params sets in |ctx| the parameters associated with
 // the signature scheme defined by the given |nid|. It returns one on success
-// and zero on error.
+// and zero on error. This API is marked as EXPERIMENTAL (using the deprecated
+// warning) to indicate that this API may change as the standards around the
```

Comment:
Functions not actually marked as experimental/deprecated.

Reply:
What's the right mechanism for this? `OPENSSL_DEPRECATED`? Or should we create a similar macro `OPENSSL_EXPERIMENTAL`?

Reply:
Andrew and I considered the use of an alias OPENSSL_EXPERIMENTAL that is set to indicate these are experimental, but without a plan in mind on when and how we plan to remove such a flag, I worry that we are just swapping one flag out for a form of another. Is there something from preventing us from reaching a stable API? The ML-KEM experimental functions don't use such a flag for example, and are placed in a different header file.